app/Http/Controllers/LabelsController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Asset;
+use App\Models\AssetModel;
+use App\Models\Category;
+use App\Models\Company;
+use App\Models\Labels\Label;
+use App\Models\Manufacturer;
+use App\Models\Setting;
+use App\Models\User;
+use App\View\Label as LabelView;
+use Illuminate\Support\Facades\Storage;
+
+class LabelsController extends Controller
+{
+    /**
+     * Returns the Label view with test data
+     *
+     * @author Grant Le Roux <grant.leroux+snipe-it@gmail.com>
+     * @param  string  $labelName
+     * @return \Illuminate\Contracts\View\View
+     */
+    public function show(string $labelName)
+    {
+        $this->authorize('view', Label::class);
+
+        $labelName = str_replace('/', '\\', $labelName);
+        $template = Label::find($labelName);
+
+        $this->authorize('view', $template);
+
+        $testAsset = new Asset();
```

Comment:
Again, TINY nit - can we rename this to something a little clearer, like `$exampleAsset` or `$previewAsset`?

Reply:
i have a branch on my fork that renames this to `$exampleAsset` in case that's useful in helping get this merged